test/clojure/core_test/juxt.cljc

```diff
@@ -0,0 +1,42 @@
+(ns clojure.core-test.juxt
+  (:require [clojure.test :as t :refer [deftest testing is are]]
+            [clojure.core-test.portability #?(:cljs :refer-macros :default :refer) [when-var-exists]]))
+
+(when-var-exists juxt
+  (deftest test-juxt
+    (testing "'returns a function'"
+      (is (ifn? (juxt juxt partial comp :foo)))
+      (is (ifn? (juxt nil)))
+      (is (ifn? (juxt (range)))))
+
+    (testing "'the returned fn...returns a vector'"
+      (is (vector? ((juxt inc) 5))))
+
+    (testing "'The returned fn takes a variable number of args'"
+      (is (= ["12345678910" 1 10]
+             ((juxt str min max) 1 2 3 4 5 6 7 8 9 10))))
+
+    (testing "'returns a fn that is the juxtaposition of those fns'"
```

Comment:
Thanks, Dave. The tests look good. One oddity stands out, though.

These four `testing` forms have double insulation of quoting. Was that intentional? I don't think we're using that anywhere else.